src/HomeScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import { ScrollView, StatusBar, View } from 'react-native';
+import { Button } from '@bluebase/components';
+import React from 'react';
+// tslint:disable: jsx-no-lambda max-line-length
+
+export class HomeScreen extends React.Component<any> {
```

Comment:
what is the purpose of this component?

Reply:
i just removed the component as it was not helpful